core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java

```diff
@@ -304,6 +261,80 @@ private void discretizePathIterator(
         }
     }
 
+    @SuppressWarnings({"checkstyle:linelength"})
```

Comment:
This checkstyle suppresss warning must not be set. Please remove it and fix the problem.